package.json

```diff
@@ -6,26 +6,23 @@
   "module": "lib-esm/index.js",
   "exports": {
     ".": {
-      "node": "./lib/index.js",
       "require": "./lib/index.js",
-      "default": "./lib-esm/index.js"
+      "import": "./lib-esm/index.js",
```

Comment:
JSON doesn't allow a trailing comma.

Reply:
Ah, the danger of trusting myself with GitHub's web editor.

Thanks, I'll get that sorted.